src/unity/schemas/Account.yml

```diff
@@ -13,4 +13,13 @@ properties:
   balance:
     type: number
     description: remaining balance on the account, nil if none
-required: [id, balance, type]
+  billingContact:
+    $ref: './BillingContact.yml'
+    description: billing contact for the account
+  users:
+    type: array
+    $ref: './Users.yml'
+  marketplaceSubscription:
+    $ref: './MarketplaceSubscription.yml'
+    description: which marketplace, nil if none
```

Comment:
added these back as part of the Account type since we only ever reference the Account in the Operator pages